packages/docusaurus-plugin-content-blog/src/feed.ts

```diff
@@ -54,11 +55,35 @@ async function generateBlogFeed({
     copyright: feedOptions.copyright,
   });
 
+  const feedItems = await (feedOptions.createFeedItems
```

Comment:
maybe simplify with:

```js
const createFeedItems = feedOptions.createFeedItems ?? defaultCreateFeedItems
```

?

Reply:
Would love to but the signatures are subtly different.  Did ponder other simplifications but didn't come up with anything that seemed worthwhile.  Happy to implement any other ideas?

Reply:
Would it be a big problem if the 2 functions had the exact same signature? It shouldn't harm anyone if defaultCreateFeedItems would "receive itself" as a param, although it wouldn't be used in practice.

Reply:
gave it a try and apparently TS is fine with that thanks to duck typing 👍 updated the PR